spec/que/views/que_jobs_ext_spec.rb

```diff
@@ -0,0 +1,200 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+if Que.db_version >= 8
+  describe "Que Jobs Ext View" do
+    class TestJob < Que::Job
+      include Que::JobMethods
+
+      def default_resolve_action
+        # prevents default deletion of complete jobs for testing purposes
+        finish
+      end
+
+      def run 
+        sleep(0.1)
```

Comment:
I'd like to avoid making the tests take longer from adding sleeps. How about using a mutex instead?